docs/docs/getting-started/installation/aws.md

```diff
@@ -128,15 +128,68 @@ Error creating vpc:  operation error EC2: CreateVpc, https response error Status
+
+Create a self-signing certificate (skip step if you already have your existing https certificate).
+The certificates will be created in your `~/.airy/certs` directory, using the OpenSSL utility.
+
+```sh
+mkdir -p ~/.airy/certs/
+cd ~/.airy/certs/
+openssl genrsa 2048 > private.key
+openssl req -new -x509 -nodes -sha1 -days 3650 -extensions v3_ca -key private.key > public.crt
+-----
+Country Name (2 letter code) [DE]:DE
+State or Province Name (full name) [Berlin]:Berlin
+Locality Name (eg, city) []:Berlin
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:Airy GmbH
+Organizational Unit Name (eg, section) []:Development
+Common Name (e.g. server FQDN or YOUR name) []:*.elb.amazonaws.com
+Email Address []:sre@airy.co
+```
+
+Note that when generating the public certificate `public.crt` you must specify a common namd of fqdn, otherwise the certificate will not be used by the AWS LoadBalancer.
+
+Next you should upload your certificates to the AWS certificates manager (ACM).
+
+```sh
+aws acm import-certificate --certificate fileb://public.crt --private-key fileb://private.key --region us-east-1
+```
+
+In case you have your own certificates and have one more file with the certificate chain, you can also specify that file as anadditional `--certificate` value.
+
+Get the address of your LoadBalancer, assuming that you have your `kube.conf` file in the current working directory:
+
+```sh
+lb=$(kubectl --kubeconfig ./kube.conf  get service traefik -n kube-system -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+```
+
+Modify the existing ingress service to reconfigure the AWS LoadBalancer.
+
+```sh
+kubectl --kubeconfig ./kube.conf -n kube-system annotate service traefik "service.beta.kubernetes.io/aws-load-balancer-backend-protocol=http"
+kubectl --kubeconfig ./kube.conf -n kube-system annotate service traefik "service.beta.kubernetes.io/aws-load-balancer-ssl-cert=${lb}"
+kubectl --kubeconfig ./kube.conf -n kube-system annotate service traefik "service.beta.kubernetes.io/aws-load-balancer-ssl-ports=https"
+kubectl --kubeconfig ./kube.conf -n kube-system patch service traefik --patch '{"spec": { "type": "LoadBalancer", "ports": [ { "name": "https", "port": 443, "protocol": "TCP", "targetPort": 80 } ] } }'
+```
+
+Then the frontend and the API services of `Airy Core` will be accessible through https on the URL of the loadbalancer:
+
+```sh
+kubectl --kubeconfig ./kube.conf -n kube-system get service traefik --output jsonpath='https://{.status.loadBalancer.ingress[0].hostname}{"\n"}'
+```
+
 ## Integrate public webhooks
 
 The public webhooks will be accessible on the public LoadBalancer which is
 created by the Ingress loadBalancer Kubernetes service.
 
-To get the public URL of your AWS Airy Core installation you can run:
+To get the public URL of your AWS Airy Core installation, asuming that your `kube.conf` file is in the current working directory, you can run:
```

Comment:
Using an ENV dir for the kube conf in the snippets, we also avoid adding this sentence before each example